### ISSUE TITLE
Rerf can work for continuous y

### ISSUE THREAD

**Issue by cshen6** [neurodata/lumberjack#71]
say [x,y] from joint normal, then do RerF(x,y,trees=500), it outputs normally.

Jovo says it should not output unless y is categorical.

**Comment by jovo** [comment]
@MrAE @falkben @ttomita @jbrowne6 should lumberjack work currently if y is continuous?

**Comment by jbrowne6** [comment]
Y should be modifiable to a factor. So Y could be an integer but each unique value will be considered a class.

James.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jbrowne6** [comment]
A quick caveat:

The version of RerF with regression implemented will take a continuous Y but I’m not sure if that’s available yet.

James.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jovo** [comment]
@jbrowne6 i don't understand your answer.  Y is continuous, given that as input, i'd think it would spit out that it doesn't support regression yet (until regression is implemented).  it seems it is not doing that?
